bin/src/named.rs

```diff
@@ -679,7 +613,7 @@ fn config_https(
 
 #[cfg(feature = "dns-over-quic")]
 fn config_quic(
-    args: &Args,
+    args: &Cli,
```

Comment:
Mostly due to name clash with clap own `Args` can change to `Opts` if it's clearer. Or since we don't use clap `Args` anymore revert.